### ISSUE TITLE
Missing ) after arguement list error

### ISSUE THREAD

**Issue by rafael-gal** [guybedford/es-module-shims#400]
I've come across es-module-shims because I wanted to add <script type="importmap"></script> on an eletron app that has lower chromium version

![image](https://github.com/guybedford/es-module-shims/assets/77764129/c567a16b-c563-48da-a26e-d82cc67c04be)

I follow through [Usage](https://github.com/guybedford/es-module-shims#usage) but I couldn't find a documentation or issue that relates to mine.

![image](https://github.com/guybedford/es-module-shims/assets/77764129/9651a269-5d41-4388-911e-e50e889955d7)

for the record I've tried adding simple module script like

```
<script type="module">
import { Component } from 'preact'
console.log('Component', Component)
</script>
```
It does work and shows loggings so it somehow runs but from my /app.js it shows the error

**Comment by rafael-gal** [comment]
it turns out it's a nested dynamic-import issue same issue at [#275](https://github.com/guybedford/es-module-shims/issues/275) I was able to resolve the issue by doing 

`
const __META__ = import.meta
const MetaURL = ___META___.url
const { name: ModuleName } = __MODULE__(MetaURL)
`

some issues still persist on ShimMode ON

Blank Display
![image](https://github.com/guybedford/es-module-shims/assets/77764129/8bdca94c-c5d2-4da6-bb14-29d271bd328b)

ShimMode Off
Blank Display + 2 Relative references Error
![image](https://github.com/guybedford/es-module-shims/assets/77764129/f2bce4f3-dc19-462b-960e-19376b3157a9)

**Comment by rafael-gal** [comment]
Disregard my blank page. I was Adding my router in window.addEventListener( 'load' ) and it's not being triggered not even once.

**Comment by guybedford** [comment]
The specifier error is correct in polyfill mode, as the native loader has to fail before the polyfill kicks in. In both scenarios it seems you have a blank page?

It would be great if you could share the bug / error that points to `import.meta` as the issue here, and also share a replication for looking into it further.

**Comment by rafael-gal** [comment]
> The specifier error is correct in polyfill mode, as the native loader has to fail before the polyfill kicks in. In both scenarios it seems you have a blank page?

Yes, both page werre blank in both modes. It was my bad I was unsure of the problem at first but after using shim mode it points out the specific line where it failed. Its's all good.

> It would be great if you could share the bug / error that points to `import.meta` as the issue here, and also share a replication for looking into it further.

I may have to get back with you for that and try If I can reproduce the issue on sandbox.
